src/ui/CozyTheme/CozyTheme.tsx

```diff
@@ -10,17 +10,22 @@ interface CozyThemeState {
   variant: CozyThemeVariant
 }
 
-export const CozyThemeContext = createContext<CozyThemeState>({
+// We'll consider the context as possibly undefined
+// This will allow us to use the runtime hook with a fallback value and correctly type it
+export const CozyThemeContext = createContext<CozyThemeState | undefined>({
   variant: 'normal'
 })
 
+// We'll let the user choose the variant or use the implied default one
+// In effect the variant is not optional in the actual context value
 interface CozyThemeProps {
-  variant: CozyThemeVariant
+  variant?: CozyThemeVariant
   children: ReactNode
 }
 
+// Will default to `normal` variant if none is provided
 export const CozyTheme = ({
-  variant,
+  variant = 'normal',
```

Comment:
I would prefer to have this not optional, as I find `<CozyTheme variant="normal">` easier to understand when reading the code compared to `<CozyTheme>` and the gain is significant enough to justify it.

Reply:
✅